### ISSUE TITLE
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './cid' is not defined by "exports" in /node_modules/multiformats/package.json

### ISSUE THREAD

**Issue by thinkcucumber** [orbitdb/orbitdb#1023]
I am using Node.js 16.14.2, npm 8.19.2.

It seems like module 'orbit-db' is a CommonJS module. So I have the following simple Node.js script:

_test.js_

`const OrbitDB = require('orbit-db')`

Which is taken from your documentation here:

[module-with-ipfs-instance](https://github.com/orbitdb/orbit-db#module-with-ipfs-instance)

I am trying to run it like this:

`$ node test.js`

But I'm getting the following error:

```
$ node test.js 
node:internal/modules/cjs/loader:488
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './cid' is not defined by "exports" in /node_modules/multiformats/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:453:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:671:7)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/node_modules/orbit-db/src/orbit-db-address.js:3:17) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

Orbitdb uses the multiformats module which seems to be an ES module. So how can I require/import the Orbitdb module?

**Comment by anarkrypto** [comment]
I was able to fix by dowgrading multiformats to v9.9.0

```bash
rm node_modules/multiformats
yarn add multiformats@9.9.0
```

**Comment by modenero** [comment]
> I was able to fix by dowgrading multiformats to v9.9.0
> 
> ```shell
> rm node_modules/multiformats
> yarn add multiformats@9.9.0
> ```

this worked for me as well

**Comment by haadcode** [comment]
This should be fixed in 1.0.0. Note the [new package name](https://www.npmjs.com/package/@orbitdb/core).

If the issue persists, feel free to re-open the issue.
